### PR TITLE
toc restructure 2

### PR DESCRIPTION
**Pull request by davidmu1**

### CODE REVIEW

File: api-reference/beta/resources/report-identity-access.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Get a better understanding of how users in your organization use Azure AD capabilities, such as self-service password rest and multi-factor authentication (MFA). You can determine which authentication methods are more successful for your organization, what types of errors end users are running into, and what campaign you need to run to help your end users adopt the use of self-service password rest and MFA. For more information, see the [authentication methods usage report API](/graph/api/resources/applicationsigninsummary?view=graph-rest-beta).
+
+## Next steps
+
+Reports resources and APIs can open up new ways for you to engage with users and manage their experiences with Microsoft Graph. To learn more:
+
+- Drill down on the methods and properties of the resources most helpful to your scenario.
+- Try the API in the [Graph Explorer](https://developer.microsoft.com/graph/graph-explorer).
+
+Need more ideas? See [how some of our partners are using Microsoft Graph](https://developer.microsoft.com/graph/graph/examples#partners).
```
Comment (Lauragra) [inline api-reference/beta/resources/report-identity-access.md:47]: None of the partners are using reporting scenarios currently. Do you still want to link to the partners page? Also, we have an updated URL: https://developer.microsoft.com/en-us/graph/partners

Comment (davidmu1) [reply api-reference/beta/resources/report-identity-access.md]: Thanks Laura. Removed the link.

File: api-reference/beta/toc.yml [hunk 2]
```diff
@@ -10940,8 +10896,33 @@ items:
             href: api/multivaluelegacyextendedproperty-get.md
     - name: Reports
       items:
+      - name: Azure AD activity reports
```
Comment (Lauragra) [inline api-reference/beta/toc.yml:10899]: Just to verify, is this the structure that you want for the beta TOC?
image

Comment (davidmu1) [reply api-reference/beta/toc.yml:10899]: Yes, I thought it was consistent with what is there.

Comment (Lauragra) [reply api-reference/beta/toc.yml:10899]: Okay, I didn't know if the Azure AD node should be placed under Identity and access, rather than as a peer.

Comment (davidmu1) [reply api-reference/beta/toc.yml:10899]: I guess I don't feel strongly either way. If you think they should be moved under Identity and Access I'm fine with that. They were under the Identity and Access node and Dan suggested they would be better in under reports. Maybe Azure AD activity reports isn't the right title, just was looking for something more inclusive than Audit logs.

**Pull request merged**